File: step07customview/src/main/java/com/gura/step07customview/MainActivity.java
package com.gura.step07customview;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
//        setContentView(R.layout.activity_main);
        setContentView(new TouchView(this));    //Layout으로 화면을 채우는 것이 아니라 만든 View 로 화면을 채우겠다.
    }
}

File: step20hellokotlin/src/main/java/com/gura/step20hellokotlin/MainActivity.kt
package com.gura.step20hellokotlin

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {    // 물음표는 null 값도 허용한다는 의미
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }
}
